### PR TITLE
adjust rem weight formula

### PR DESCRIPTION
**Pull request by Art17**
adjust rem voting formula

### CODE REVIEW

File: unittests/rem_voting_test.cpp [hunk 1]
```diff
@@ -489,7 +489,7 @@ BOOST_FIXTURE_TEST_CASE( rem_vote_weight_test, voting_tester ) {
          // rem weight:      0.72;
```
Comment (roman-tik) [inline unittests/rem_voting_test.cpp]: update "weeks to mature" and "rem weight" in the comments

File: unittests/rem_voting_test.cpp [hunk 2]
```diff
@@ -510,20 +510,20 @@ BOOST_FIXTURE_TEST_CASE( rem_vote_weight_test, voting_tester ) {
          // rem weight:      0.678;
```
Comment (roman-tik) [inline unittests/rem_voting_test.cpp]: update "weeks to mature" and "rem weight" in the comments

File: unittests/rem_voting_test.cpp [hunk 3]
```diff
@@ -510,20 +510,20 @@ BOOST_FIXTURE_TEST_CASE( rem_vote_weight_test, voting_tester ) {
...
 
       // 8 Weeks Later
       {
          produce_min_num_of_blocks_to_spend_time_wo_inactive_prod(fc::days( 7 * 8 )); // +8 weeks
          votepro( N(whale1), { N(proda) } );
 
-         // weeks to mature: 0;
+         // weeks to mature: 0;1.3925604968379901e+18
          // eos weight:      1.811133596417372e+06;
          // rem weight:      1.000000;
```
Comment (roman-tik) [inline unittests/rem_voting_test.cpp]: update "rem weight" in the comments

File: unittests/rem_voting_test.cpp [hunk 4]
```diff
@@ -510,20 +510,20 @@ BOOST_FIXTURE_TEST_CASE( rem_vote_weight_test, voting_tester ) {
...
+         BOOST_TEST_REQUIRE( 8.4652838071982426e+17 == prod["total_votes"].as_double() );
       }
 
       // 8 Weeks Later
       {
          produce_min_num_of_blocks_to_spend_time_wo_inactive_prod(fc::days( 7 * 8 )); // +8 weeks
          votepro( N(whale1), { N(proda) } );
 
-         // weeks to mature: 0;
+         // weeks to mature: 0;1.3925604968379901e+18
```
Comment (roman-tik) [inline unittests/rem_voting_test.cpp]: "weeks to mature" is an integer, not double

File: unittests/rem_voting_test.cpp [hunk 5]
```diff
@@ -510,20 +510,20 @@ BOOST_FIXTURE_TEST_CASE( rem_vote_weight_test, voting_tester ) {
...
          // eos weight:      1.811133596417372e+06;
          // rem weight:      1.000000;
          // staked:          799999999000
          const auto prod = get_producer_info( name("proda") );
-         BOOST_TEST_REQUIRE( 1.4489068753227643e+18 == prod["total_votes"].as_double() );
+         BOOST_TEST_REQUIRE( 1.3925604968379901e+18 == prod["total_votes"].as_double() );
 
       }
```
Comment (roman-tik) [inline unittests/rem_voting_test.cpp]: add one more test where "weeks to mature" = 0

**Review by roman-tik — CHANGES_REQUESTED** [review]

File: unittests/rem_voting_test.cpp (hunk 1 above)
Comment (Art17) [reply unittests/rem_voting_test.cpp]: Done.

File: unittests/rem_voting_test.cpp (hunk 2 above)
Comment (Art17) [reply unittests/rem_voting_test.cpp]: Done.

File: unittests/rem_voting_test.cpp (hunk 3 above)
Comment (Art17) [reply unittests/rem_voting_test.cpp]: Done.

File: unittests/rem_voting_test.cpp (hunk 4 above)
Comment (Art17) [reply unittests/rem_voting_test.cpp]: Done.

File: unittests/rem_voting_test.cpp (hunk 5 above)
Comment (Art17) [reply unittests/rem_voting_test.cpp]: Done.

File: unittests/rem_voting_test.cpp [hunk 6]
```diff
@@ -393,140 +393,154 @@ BOOST_FIXTURE_TEST_CASE( rem_voting_test, voting_tester ) {
...
+        {
+            produce_min_num_of_blocks_to_spend_time_wo_inactive_prod(fc::seconds(30 * 24 * 3600)); // +30 days
+
+            const auto r = delegate_bandwidth(N(rem.stake), N(whale1), asset(20'000'000'0000LL));
+            BOOST_REQUIRE( !r->except_ptr );
+
+            votepro( N(whale1), { N(proda) } );
+
+            // adjusted:   now + 30 Days * 60 / 80 + 180 Days * 20 / 80 => 67.5 Days
+            // weeks to mature: 9;
```
Comment (roman-tik) [inline unittests/rem_voting_test.cpp]:
```suggestion
            // weeks to mature: 4;
```

File: unittests/rem_voting_test.cpp [hunk 7]
```diff
@@ -393,140 +393,154 @@ BOOST_FIXTURE_TEST_CASE( rem_voting_test, voting_tester ) {
...
+            // eos weight:      1.627939195726894e+06;
+            // rem weight:      0.65;
```
Comment (roman-tik) [inline unittests/rem_voting_test.cpp]:
```suggestion
            // rem weight:      0.844444;
```

**Review by roman-tik — CHANGES_REQUESTED** [review]

File: unittests/rem_voting_test.cpp [hunk 8]
```diff
@@ -393,140 +393,154 @@ BOOST_FIXTURE_TEST_CASE( rem_voting_test, voting_tester ) {
...
+        }
 
-      // Day 210 (30)
-      // re-staking vote power 83%
-      // staked 60KK
-      // re-staked 20KK
-      {
-         produce_min_num_of_blocks_to_spend_time_wo_inactive_prod(fc::seconds(30 * 24 * 3600)); // +30 days
+        // Day 210 (30)
+        // re-staking vote power 83%
```
Comment (roman-tik) [inline unittests/rem_voting_test.cpp]:
```suggestion
        // re-staking vote power 84%
```

File: unittests/rem_voting_test.cpp (hunk 6 above)
Comment (Art17) [reply unittests/rem_voting_test.cpp]: Done.

File: unittests/rem_voting_test.cpp (hunk 7 above)
Comment (Art17) [reply unittests/rem_voting_test.cpp]: Done.

File: unittests/rem_voting_test.cpp (hunk 8 above)
Comment (Art17) [reply unittests/rem_voting_test.cpp]: Done.

**Review by roman-tik — APPROVED** [review]

**Pull request merged**